src/sidebar/components/LaunchErrorPanel.js

```diff
@@ -0,0 +1,24 @@
+import Panel from './Panel';
+
+/**
+ * @typedef LaunchErrorPanelProps
+ * @prop {Error} error - The error that prevented the client from launching
+ */
+
+/**
+ * An error panel displayed when a fatal error occurs during app startup.
+ *
+ * Note that this component cannot use any of the services or store that are
```

Comment:
Thanks for this comment. Could really save hair-tearing later. As an aside, I think this constraint validates creating a separate component for this, otherwise I would have suggested doing something more generic here...